src/Raven.Server/Documents/ETL/Providers/Elasticsearch/ElasticSearchLoadException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
```

Comment:
I have just noticed that the directory name is still "Elasticsearch". See  `src/Raven.Server/Documents/ETL/Providers/Elasticsearch/ElasticSearchLoadException.cs`

Reply:
done